src/main/groovy/lang/ObjectRange.java

```diff
@@ -537,8 +539,9 @@ protected Object increment(Object value) {
      * @param value the value to decrement
      * @return the decremented value
      */
-    protected Object decrement(Object value) {
-        return InvokerHelper.invokeMethod(value, "previous", null);
+    @SuppressWarnings("unchecked")
+    private T decrement(T value) {
```

Comment:
This represents a binary incompatible change. We try really hard to avoid those - it leads to lots of problems, e.g. classes compiled with one version of Groovy won't work with other versions.